waku/node/waku_node.nim

```diff
@@ -249,12 +287,19 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
 
     await node.wakuArchive.handleMessage(topic, msg)
 
+  proc syncHandler(topic: PubsubTopic, msg: WakuMessage) =
```

Comment:
Just a question, is there any particular reason it cannot be async? (or just because it is call ~sync~Handler :-) ?)

Reply:
:thinking: I assumed that async was not needed but maybe it would be better so that not blocking can occurs.